tests/QuantityValueTests.cs

```diff
@@ -233,6 +233,20 @@ public void Serialize()
                 ResourceQuantity quantity = 12000;
                 Assert.Equal("\"12e3\"", JsonConvert.SerializeObject(quantity));
             }
+        }
+
+        [Fact]
+        public void DeserializeYaml()
+        {
+            var value = Yaml.LoadFromString<ResourceQuantity>("\"1\"");
+            Assert.Equal(new ResourceQuantity(1, 0, DecimalSI), value);
+        }
+
+        [Fact]
+        public void SerializeYaml()
+        {
+            var value = Yaml.SaveToString(new ResourceQuantity(1, -1, DecimalSI));
+            Assert.Equal("100m\r\n...\r\n", value);
```

Comment:
`BuildValueSerializer` will not emit document end.
or
`Environment.NewLine` ?

Reply:
I changed the code to use `BuildValueSerializer()`